src/libcollectionstest/bit/vec.rs

```diff
@@ -630,6 +630,110 @@ fn test_bit_vec_extend() {
                                  0b01001001, 0b10010010, 0b10111101]));
 }
 
+#[test]
+fn test_bit_vec_append() {
+    // Append to BitVec, that holds a multiple of u32::BITS bits
+    let mut a = BitVec::from_bytes(&[0b10100000, 0b00010010, 0b10010010, 0b00110011]);
+    let mut b = BitVec::new();
+    b.push(false);
+    b.push(true);
+    b.push(true);
+
+    a.append(&mut b);
+
+    assert_eq!(a.len(), 35);
+    assert_eq!(b.len(), 0);
+    assert!(b.capacity() >= 3);
```

Comment:
According to [RFC 509](https://github.com/rust-lang/rfcs/blob/master/text/0509-collections-reform-part-2.md#add-the-following-methods), it should be possible to reuse the container on the right-hand side.